inc/ooo_cpu.h

```diff
@@ -15,7 +15,7 @@ using namespace std;
 // CORE PROCESSOR
 #define FETCH_WIDTH 6
 #define DECODE_WIDTH 6
-#define EXEC_WIDTH 6
+#define EXEC_WIDTH 4
```

Comment:
Why are we decreasing execution width here? This would create a bottleneck.

Reply:
The idea here is that the default config approximately matches an Intel Sunny Cove core, which has 4 non-AGU ALUs, hence the 4-wide execution.  I had previously set it to 6 in an attempt to compensate for how I believed the simulator worked, which was wrong.  I believed that instructions with loads would have to go through execute twice,- once when their registers were ready, and again when their loads were ready, but really they were going through execute zero (0) times.  After this fix, all instructions now respect the EXEC_WIDTH limits at the complete_execution() phase, so making the EXEC_WIDTH 4 now matches the original intention.